tests/integration/test_security_report.py

```diff
@@ -126,14 +126,19 @@ def test_security_report(mini_sentry, relay, test_case, json_fixture_provider):
 
     assert resp.status_code == 200
 
-    envelope = mini_sentry.captured_events.get(timeout=1)
+    envelope = mini_sentry.captured_events.get(timeout=7)
     event = get_security_report(envelope)
     for x in ignored_properties:
         event.pop(x, None)
 
     ext = ".no_processing.output"
     expected_evt = fixture_provider.load(test_name, ext)
 
+    if "received" in event:
+        event.pop("received")
+    if "timestamp" in event:
+        event.pop("timestamp")
+
```

Comment:
At least in the `expect_ct` case the event has the `received` and `timestamp` fields. I haven't thought if we want them, but I don't think this is that important right now and I took the short and hacky path deleting these fields here. We may want to revisit this.

Reply:
I think this is fine, just a result of timestamp normalization.